playbooks/azure/openshift-cluster/tasks/create_blob_from_vm.yml

```diff
@@ -23,7 +23,7 @@
 - name: start copy
   command: >
     az storage blob copy start
-    --source-uri "{{ (sas.stdout | from_json).properties.output.accessSAS }}"
+    --source-uri "{{ (sas.stdout | from_json).accessSas }}"
```

Comment:
This has gone back and forth a couple of times.  We just need to ensure this works inside of the origin-ansible image.

Reply:
yeah I don't know if the output is different on CI but here it sure returns a simpler structure

Reply:
We anchor the version of azure cli so that the output doesn't change since this has bitten us in the past.

https://github.com/openshift/openshift-ansible/blob/master/images/installer/Dockerfile#L13

https://github.com/openshift/openshift-ansible/blob/master/playbooks/azure/openshift-cluster/tasks/create_blob_from_vm.yml#L18

I would verify that your version in your pip install or rpm matches `azure-cli-2.0.46`.  This will guarantee us that we maintain the same consistent output.

Reply:
@kwoodson understood, reverted that change in 268cebe6c